include/GafferUI/StandardStyle.h

```diff
@@ -91,6 +92,7 @@ class GAFFERUI_API StandardStyle : public Style
 		void renderConnection( const Imath::V3f &srcPosition, const Imath::V3f &srcTangent, const Imath::V3f &dstPosition, const Imath::V3f &dstTangent, State state = NormalState, const Imath::Color3f *userColor = nullptr ) const override;
 		Imath::V3f closestPointOnConnection( const Imath::V3f &p, const Imath::V3f &srcPosition, const Imath::V3f &srcTangent, const Imath::V3f &dstPosition, const Imath::V3f &dstTangent ) const override;
 		void renderAuxiliaryConnection( const Imath::Box2f &srcNodeFrame, const Imath::Box2f &dstNodeFrame ) const override;
+		void renderAuxiliaryNode( float radius, char label, State state, const Imath::Color3f *userColor ) const override;
```

Comment:
I was kinda thinking of taking a shortcut here, and using the same trick that DotNodeGadget uses to get the circles. Not against this though if Andrew's happy to bump major version soon : it definitely gives more scope to the Style to style things. On the question of scope, perhaps a `const Box2f &bound` is less prescriptive, allowing the style to use circles if it wants, but also rectangles or something else if not? And perhaps the labelling should be a separate thing from the frame, as it is for StandardNodeGadget and `renderNodeFrame()`? i.e. We could just use `renderText()` from within `AuxiliaryNodeGadget::doRenderLayer()`?